src/app/modules/timesheet/services/timesheet.service.ts

```diff
@@ -120,14 +120,44 @@ export class TimesheetService {
   }
 
   public openLastTimesheetInLocal(): boolean {
-    const timesheetsOfLocalStorage = this.getTimesheetsLocal();
+    const timesheetsOfLocalStorage = this.getLocalstorageTimesheetsList();
     if (timesheetsOfLocalStorage.length > 0) {
       this.setTimesheet(this._localSaveService.getLocalItem(timesheetsOfLocalStorage[timesheetsOfLocalStorage.length - 1]));
       return true;
     }
     return false;
   }
 
+  public getPresentTimesheet(transferedTimesheet: Timesheet) {
+    const timesheetsOfLocalStorage = this.getLocalstorageTimesheetsList();
+    const localStorageSize = this.getLocalstorageTimesheetsList().length;
+    for (let i = localStorageSize; i >= 0; i--) {
+      const transfertTimesheet: Timesheet = Object.assign({},
+                                                          new Timesheet(),
+                                                          this._localSaveService.getLocalItem(timesheetsOfLocalStorage[i]));
+
```

Comment:
Indent like that :
```
const transfertTimesheet: Timesheet = Object.assign(
     {},
     new Timesheet(),
     this._localSaveService.getLocalItem(timesheetsOfLocalStorage[i])
);
```

Reply:
Soon you won't be able to do things like that anymore ! Muahaha 😄

Reply:
you speak about that ? https://github.com/Iteatime/Acrabadabra/tree/roliva-add-prettier-and-some-rules

Reply:
Yes but it's verrrry long to adapt all actual code to pass the linter :(